h/services/search_index/_queue.py

```diff
@@ -162,40 +162,54 @@ def sync(self, limit):
             if job.kwargs.get("force", False):
                 annotation_ids_to_sync.add(annotation_id)
                 job_complete.append(job)
-                counts[Queue.Result.SYNCED_FORCED.format(tag=job.tag)] += 1
-                counts[Queue.Result.SYNCED_TAG_TOTAL.format(tag=job.tag)] += 1
-                counts[Queue.Result.SYNCED_TOTAL] += 1
-                counts[Queue.Result.COMPLETED_FORCED.format(tag=job.tag)] += 1
-                counts[Queue.Result.COMPLETED_TAG_TOTAL.format(tag=job.tag)] += 1
-                counts[Queue.Result.COMPLETED_TOTAL] += 1
+                counts[Queue.Result.SYNCED_FORCED.format(tag=job.tag)].add(
+                    annotation_id
+                )
+                counts[Queue.Result.SYNCED_TAG_TOTAL.format(tag=job.tag)].add(
+                    annotation_id
+                )
+                counts[Queue.Result.SYNCED_TOTAL].add(annotation_id)
+                counts[Queue.Result.COMPLETED_FORCED.format(tag=job.tag)].add(job.id)
+                counts[Queue.Result.COMPLETED_TAG_TOTAL.format(tag=job.tag)].add(job.id)
+                counts[Queue.Result.COMPLETED_TOTAL].add(job.id)
```

Comment:
For the various `SYNCED` metrics we add annotation IDs to the sets, so the same annotation won't get double counted.

But for the `COMPLETED` metrics we add job IDs to the sets: You complete jobs, not annotations. If there were two jobs for the same annotation, and you synced that annotation to Elasticsearch, you completed two jobs. Not one. Even though you only synced one annotation.